src/goto-symex/goto_symex.h

```diff
@@ -357,17 +357,17 @@ class goto_symext
 
   virtual bool get_unwind_recursion(
     const irep_idt &identifier,
-    const unsigned thread_nr,
+    unsigned thread_nr,
     unsigned unwind);
 
   void parameter_assignments(
-    const irep_idt function_identifier,
+    irep_idt function_identifier,
```

Comment:
Shouldn't this instead become a const reference (so add the reference, and don't remove the const).